src/ident/Reference.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Ident;
+
+use Rinq\Revision;
+
+/**
+ * Reference refers to a session at a specific revision.
+ */
+class Reference
+{
+    /**
+     * Create a new Reference.
+     *
+     * @param SessionId $sessionId The unique ID session.
+     * @param Revision  $revision  A "version" of a session.
+     */
+    public static function create(
+        SessionId $sessionId,
+        Revision $revision
```

Comment:
`Revision` here is referring to the wrong type. It's a bit confusing because in the go package there's a `rinq.Revision` and an `ident.Revision`. `ident.Revision` is essentially an alias for a 32-bit integer. 

Nothing in the `ident` package refers to anything from `rinq` and that should hold true for PHP too.

Because PHP doesn't have type-aliases `$revision` here should just be an int.